Comparing source compatibility of commons-fileupload-1.5.jar against commons-fileupload-1.4.jar
+++  NEW CLASS: PUBLIC(+) org.apache.commons.fileupload.FileCountLimitExceededException  (compatible)
	+++  CLASS FILE FORMAT VERSION: 50.0 <- n.a.
	+++  NEW INTERFACE: java.io.Serializable
	+++  NEW SUPERCLASS: org.apache.commons.fileupload.FileUploadException
	+++  NEW CONSTRUCTOR: PUBLIC(+) FileCountLimitExceededException(java.lang.String, long)
	+++  NEW METHOD: PUBLIC(+) long getLimit()
***  MODIFIED CLASS: PUBLIC ABSTRACT org.apache.commons.fileupload.FileUploadBase  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	+++  NEW METHOD: PUBLIC(+) long getFileCountMax()
	+++  NEW METHOD: PUBLIC(+) void setFileCountMax(long)
***  MODIFIED CLASS: PUBLIC FINAL org.apache.commons.fileupload.util.Streams  (not serializable)
	===  CLASS FILE FORMAT VERSION: 50.0 <- 50.0
	***  MODIFIED FIELD: PUBLIC (<- PRIVATE) STATIC FINAL int DEFAULT_BUFFER_SIZE
